lib/k8s/client/imperative.ex

```diff
@@ -0,0 +1,734 @@
+defmodule K8s.Client.Imperative do
```

Comment:
This file is a strict move of all of the previous content from `client.ex`, yes?

Reply:
Not strict and all of the previous content, but mostly. As I wrote, here I'm not sure I understood correctly what was suggested.